jujugui/static/gui/src/app/components/scale-service/_scale-service.scss

```diff
@@ -0,0 +1,51 @@
+.scale-service {
+
+    input[type=text] {
+        margin: 20px;
+        height: 28px;
+    }
+
+    &__units {
+        position: relative;
+
+        input[type=text] {
+            background-color: transparent;
+        }
+
+        span {
+            position: absolute;
+            right: 25px;
+            top: 30px;
+            color: $mid-grey;
+        }
+    }
+
+    &__selector {
+
+      input[type=radio] {
+        opacity: 1;
+        margin: 10px 0px 10px 15px;
+
+        &+label {
+            margin: 0;
+            margin-left: 0;
+        }
+      }
+
+      div:last-of-type {
+        margin-bottom: 30px;
+      }
+
+    }
+
+    &__constraints {
+
+        input[type=text] {
```

Comment:
Can we target classes over raw elements unless its a style that is app wide otherwise these are component styles. For example if the someone changes this element to `type="number"` they will have to update the styles.